streams/src/test/java/org/apache/kafka/streams/internals/metrics/ClientMetricsTest.java

```diff
@@ -125,8 +114,8 @@ public void shouldAddAliveStreamThreadsMetric() {
     public void shouldGetFailedStreamThreadsSensor() {
         final String name = "failed-stream-threads";
         final String description = "The number of failed stream threads since the start of the Kafka Streams client";
-        expect(streamsMetrics.clientLevelSensor(name, RecordingLevel.INFO)).andReturn(expectedSensor);
-        expect(streamsMetrics.clientLevelTagMap()).andReturn(tagMap);
+        Mockito.when(streamsMetrics.clientLevelSensor(name, RecordingLevel.INFO)).thenReturn(expectedSensor);
+        Mockito.when(streamsMetrics.clientLevelTagMap()).thenReturn(tagMap);
```

Comment:
From a style perspective, you can import `when` via a static import to make it more readable.